scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -10,6 +10,8 @@
            (io.kubernetes.client.models V1Pod V1ContainerStatus V1PodStatus)
            (java.net URLEncoder)))
 
+(def cook-synthetic-pod-job-uuid-label "cook-synthetic-pod-for-job-uuid")
```

Comment:
Kubernetes likes to have a naming scheme of reverse-fqdn, with dot as a separator. We're using cook.FOO elsewhere.

Reply:
Ah, thanks, will do

Reply:
Done, now using `twosigma.com/cook-scheduler-synthetic-pod-job-uuid` to match the existing `twosigma.com/cook-scheduler-job`